src/pykeen/triples/triples_numeric_literals_factory.py

```diff
@@ -76,13 +96,21 @@ def from_path(
         path: Union[str, pathlib.Path, TextIO],
         *,
         path_to_numeric_triples: Union[None, str, pathlib.Path, TextIO] = None,
+        numeric_triples_preprocessing: Optional[Union[str, Callable[[LabeledTriples], LabeledTriples]]] = None,
```

Comment:
Optional[str, X] can always be replaced with `pykeen.typing.Hint[X]` - this cleans things up immensely. Further, you might want to save this callable as its own type alias as well.